specification/attestation/data-plane/Microsoft.Attestation/preview/2022-09-01-preview/attestation.json

```diff
@@ -0,0 +1,1206 @@
+      "properties": {
+        "token": {
+          "description": "An RFC7519 JSON Web Token structure whose body is a PolicyCertificatesModificationResult object.",
+          "$ref": "#/definitions/JsonWebToken"
+        }
+      }
+    },
+    "PolicyCertificatesModificationResult": {
+      "type": "object",
+      "description": "The result of a policy certificate modification",
+      "properties": {
+        "x-ms-certificate-thumbprint": {
+          "description": "Hex encoded SHA1 Hash of the binary representation certificate which was added or removed",
+          "type": "string",
+          "x-ms-client-name": "CertificateThumbprint"
+        },
+        "x-ms-policycertificates-result": {
+          "description": "The result of the operation",
+          "type": "string",
+          "x-ms-client-name": "CertificateResolution",
+          "enum": [
+            "IsPresent",
+            "IsAbsent"
+          ],
+          "x-ms-enum": {
+            "name": "CertificateModification",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "IsPresent",
+                "description": "After the operation was performed, the certificate is in the set of certificates."
+              },
+              {
+                "value": "IsAbsent",
+                "description": "After the operation was performed, the certificate is no longer present in the set of certificates."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "StoredAttestationPolicy": {
+      "type": "object",
+      "properties": {
+        "AttestationPolicy": {
+          "description": "Policy text to set as a sequence of UTF-8 encoded octets.",
+          "type": "string",
+          "format": "base64url"
+        }
+      }
+    },
+    "TcbBaseline": {
+      "type": "object",
+      "description": "TEE specific Tcb baseline used in attestation, and baseline details including baseline identifier, release date and minimum required software version",
+      "properties": {
+        "tcbEvaluationDataNumber": {
+          "description": "A monotonically increasing sequence number changed when Intel updates the content of the TCB evaluation data set. SGX TEE specific property",
+          "type": "string"
+        },
+        "tcbReleaseDate": {
+          "description": "Date and time when the Tcb is released. SGX TEE specific property",
+          "type": "string"
+        },
+        "minimumPswLinuxVersion": {
+          "description": "Minimum Linux PSW version required to support the corresponding Tcb baseline. SGX TEE specific property",
+          "type": "string"
+        },
+        "minimumPswWindowsVersion": {
+          "description": "Minimum Windows PSW version required to support the corresponding Tcb baseline. SGX TEE specific property",
+          "type": "string"
+        }
+      }
+    },
+    "TcbBaselineResult": {
```

Comment:
Seems this list could grow a lot over time. This should probably be pageable, which means adding an [`x-ms-pageable`](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-pageable) to the operation returning it, and a `nextLink` to the properties of this object. Even if you don't page now, if it seems even _remotely_ possible in the future, you should now or this will be a breaking change later.

Reply:
There will be one entry added every 6 months. Over a decade, that means adding 20 entries to the list. Given that, does it still make sense to make it pagable?

Reply:
That's really for the service team to say. My only cautionary example from back in the VS setup days when we worked together a bit: never say never. Too many teams have said, "that will never happen" and years (or even months) later it did and they often didn't have a lot of options. If there's a remote change this list could grow large, I'd support paging now even if you don't use it initially. If you're absolutely sure it never will be a long list, it's your call.

Reply:
From the meeting, it sounded like this wasn't likely (as you said), so resolving.